pkg/cmd/pulumi/whoami.go

```diff
@@ -69,9 +78,19 @@ func newWhoAmICmd() *cobra.Command {
 		}),
 	}
 
+	cmd.PersistentFlags().BoolVarP(
+		&jsonOut, "json", "j", false, "Emit output as JSON")
+
 	cmd.PersistentFlags().BoolVarP(
 		&verbose, "verbose", "v", false,
 		"Print detailed whoami information")
 
 	return cmd
 }
+
+// WhoAmIJSON is the shape of the --json output of this command.
+type WhoAmIJSON struct {
+	User          string   `json:"user"`
+	Organizations []string `json:"organizations,omitempty"`
+	BackendURL    string   `json:"backendUrl"`
```

Comment:
Apologies, we missed this before. Would you mind calling this "URL" `json:"url"` to match the shape of `pulumi about --json | jq .backend`? Alternatively , you can re-use the backendAbout shape, although that includes an additional Name field we don't want here, so it might be easier to just rename this field.

Reply:
No worries. That sounds good to me! Pushed.